amis/build_ami.sh

```diff
@@ -25,7 +25,7 @@ public=$3
 build_date=$4
 
 available_os="centos6 centos7 alinux ubuntu1404 ubuntu1604"
-available_regions="$(aws ec2 --region us-east-1 describe-regions --query Regions[].RegionName --output text | tr '\t' ',')"
+available_regions="$(aws ec2 --region ${region} describe-regions --query Regions[].RegionName --output text | tr '\t' ',')"
```

Comment:
${region} == all is going to break line 50

Reply:
I'll default it to `us-east-1` if it's set to `all`